src/components/job/JobApproveReject.jsx

```diff
@@ -67,10 +67,14 @@ export default function JobApproveReject({ jobId }) {
             ></textarea>
           </div>
           <div className="w-full flex flex-row justify-center mt-5">
-            <label htmlFor="feedback-display" className="btn btn-primary mr-5">
+            <label htmlFor="feedback-display" className="btn btn-secondary mr-5">
```

Comment:
sry for not making it clear. here should be a normal button, just `btn` is good

Reply:
no worries, done